jupyterhub/templates/proxy/service.yaml

```diff
@@ -81,3 +81,8 @@ spec:
     {{- . | toYaml | nindent 4 }}
   {{- end }}
   {{- end }}
+  {{- with .Values.proxy.service.externalIPs }}
+  externalIPs: {{- range . }}
```

Comment:
This probably works fine in this case, but is a risky pattern since number looking strings becomes numbers like this, while using toYaml wouldnt.

Can you switch this to use {{ . | toYaml | nindent 4 }} instead? I'm on mobile and failed to provide a code suggestion spanning multiple lines.

Reply:
I pushed a commit for this